### PR TITLE
feat(agnoster): add `anaconda` env information

### PR DESCRIPTION
**Pull request by Ochoja**
## Standards checklist:

- [ x ] The PR title is descriptive.
- [ x ] The PR doesn't replicate another PR which is already open.
- [ x ] I have read the contribution guide and followed all the instructions.
- [ x ] The code follows the code style guide detailed in the wiki.
- [ x ] The code is mine or it's from somewhere with an MIT-compatible license.
- [ x ] The code is efficient, to the best of my ability, and does not waste computer resources.
- [ x ] The code is stable and I have tested it myself, to the best of my abilities.
- [ x ] If the code introduces new aliases, I provide a valid use case for all plugin users down below.

## Changes:

- [...]

## Other comments:

...

### CODE REVIEW

**Review by carlosala — APPROVED** [review]

**Comment by carlosala** [comment]
Thanks, great addition!

**Comment by mhart** [comment]
Hmmm. This has suddenly made "base" appear in *all* of my prompts. Was it intended to be shown by default, even when on the "base" environment?

File: themes/agnoster.zsh-theme [hunk 1]
```diff
@@ -227,6 +227,14 @@ prompt_virtualenv() {
   fi
 }
 
+# Conda Virtualenv
+promp_conda_virtualenv() {
```
Comment (mhart) [inline themes/agnoster.zsh-theme:231]: Suggestion: `prompt_conda_virtualenv`

**Comment by carlosala** [comment]
Maybe we could skip base, sounds great

**Comment by carlosala** [comment]
Could you make a PR?

**Comment by H-Cao** [comment]
This change makes my shell prompt behave weirdly (the conda environment name appears twice). I tried to configure conda not to change PS1 in the hope that it would drop  ```(base)``` at the front of the shell prompt. But the other one disappeared as well.

Is there any way to get around this? Or should I keep using the old theme config? Thanks!
Screenshot 2024-05-21 at 23 18 27

**Comment by carlosala** [comment]
I reverted the changes 👍🏻

**Pull request merged**